clients/py/v3io_frames/__init__.py

```diff
@@ -67,9 +67,9 @@ def Client(address='', data_url='', container='', path='', user='',
         url=data_url or env.url,
         container=container or env.container,
         path=path or env.path,
-        user=user or env.user,
-        password=password or env.password,
-        token=token or env.token,
+        user=user or env.user or environ.get('V3IO_USERNAME'),
```

Comment:
I'm not sure that having  both `V3IO_SESSION` and `V3IO_USERNAME` is a good idea. If we do decide to keep both then:
1. This code need to move to `session_from_env`
2. We should probably look for all `V3IO_*` environment variables and align them with session field (e.g. `V3IO_USER` instead of `V3IO_USERNAME`)
3. Need to be clear what has higher priority. `V3IO_SESSION` or a specific field environment variable.

Reply:
@tebeka agree we dont need both, didnt want to break the tests. the V3IO_** are now aligned w all other V3IO based environments and our cloud provisioning which is why i changed some of them